From ddd30f563ced9cd3da5b43a509543078cd226d66 Mon Sep 17 00:00:00 2001
From: Shinya Kuribayashi <shinya.kuribayashi@necel.com>
Date: Mon, 7 Sep 2009 09:23:41 +0900
Subject: [PATCH 18/18] MIPS: nec_emma: Common NEC EMMAXXX entry in Makefile

Now we have CONFIG_SOC_EMMA, arch/mips/emma/common/, and include/asm-
mips/mach-emmaxxx/ all lined up and ready to go forward.

Signed-off-by: Shinya Kuribayashi <shinya.kuribayashi@necel.com>
---
 arch/mips/Makefile |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index d1fc00f..9ee1d32 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -322,6 +322,12 @@ cflags-$(CONFIG_MIPS_SIM)	+= -Iinclude/asm-mips/mach-mipssim
 load-$(CONFIG_MIPS_SIM)		+= 0x80100000
 
 #
+# Common NEC EMMAxxx stuff
+#
+core-$(CONFIG_SOC_EMMA)		+= arch/mips/emma/common/
+cflags-$(CONFIG_SOC_EMMA)	+= -Iinclude/asm-mips/mach-emmaxxx
+
+#
 # PMC-Sierra MSP SOCs
 #
 core-$(CONFIG_PMC_MSP)		+= arch/mips/pmc-sierra/msp71xx/
-- 
1.6.3.3

